src/features/culturalSurvey/pages/CulturalSurveyQuestions.tsx

```diff
@@ -40,11 +52,16 @@ export const CulturalSurveyQuestions = ({ route }: CulturalSurveyQuestionsProps)
     setBottomChildrenViewHeight(height)
   }
 
-  if (!culturalSurveyQuestionsData) return <React.Fragment />
+  const step = route.params.step
+  const { goBack } = useGoBack(...homeNavConfig)
+  const { answers, dispatch } = useCulturalSurveyContext()
+  const currentAnswers = answers[step]
+
+  if (!culturalSurveyQuestionsData?.questions) return <React.Fragment />
```

Comment:
Ça donne quoi ce comportement visuellement ?

Reply:
alors j'ai remplacé par un throw 
`if (!culturalSurveyQuestionsData?.questions) {
    throw new Error('should have questions to show')
  }`